README.md

```diff
@@ -4,29 +4,50 @@
 
 Welcome to the Zepto API documentation source.
 
-## How does it work?
+Zepto's API Documentation is hosted at [docs.zeptopayments.com](https://docs.zeptopayments.com/) and powered by [readme](https://readme.com/). 
+
+Note that parts of the docs published at [docs.zeptopayments.com](https://docs.zeptopayments.com/) is maintained outside of this repo. For example:
+- The PayTo API reference is located in https://github.com/zeptofs/split. 
+- The [PayTo](https://docs.zeptopayments.com/docs/payto-overview) and [Getting Started](https://docs.zeptopayments.com/docs/getting-started-in-sandbox) Developer Guide is managed separately by the _Solutions & Customer Success team_. 
+- The [Changelog](https://docs.zeptopayments.com/reference/change-log) is managed separately by the _Solutions & Customer Success team_, which is prepared based on [Internal Release Notes](https://zeptoau.atlassian.net/wiki/spaces/IRN).
+
```

Comment:
Since this is a public repo, shall we omit internal URLs? 

```suggestion
Note that some parts of the docs published at [docs.zeptopayments.com](https://docs.zeptopayments.com/) are maintained outside of this repo. 

```

Reply:
I did not realise the repo was public so thanks for pointing that out! 
Yes good call removing the internal links. 

I'll move these details to some internal team documentation.